shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -18,6 +18,11 @@
 #include "flutter/shell/common/switches.h"
 #include "flutter/shell/common/thread_host.h"
 #include "flutter/shell/platform/darwin/common/command_line.h"
+#include "flutter/shell/platform/darwin/ios/framework/Source/connection_collection.h"
+#include "flutter/shell/platform/darwin/ios/framework/Source/platform_message_response_darwin.h"
+#include "flutter/shell/platform/darwin/ios/framework/Source/profiler_metrics_ios.h"
+#include "flutter/shell/platform/darwin/ios/ios_surface.h"
+#include "flutter/shell/platform/darwin/ios/platform_view_ios.h"
 #include "flutter/shell/platform/darwin/ios/rendering_api_selection.h"
 #include "flutter/shell/profiling/sampling_profiler.h"
 
```

Comment:
Is the separate groupings something clang-format did? IIRC for Chromium the #include and #import statements were intermixed freely, just sorted by path.

Reply:
That was me; I didn't find good guidance in the style guide on that specific issue so made a guess. I can re-merge these.

Reply:
Done.